vendor/emscripten.wake

```diff
@@ -21,40 +22,45 @@ def emccFn fn = match (whichInEnvPath "emcc")
     None = Nil
     Some x = fn x
 
+
 def emppFn fn = match (whichInEnvPath "em++")
     None = Nil
     Some x = fn x
 
+
 def emmake = whichInEnvPath "emmake"
 
-def emscriptenCFlags =
-    "-Wall", "-O3", Nil
+
+def emscriptenCFlags = "-Wall", "-O3", Nil
+
 
 def emscriptenLFlags =
     "-O3",
-    "-s", "ALLOW_MEMORY_GROWTH",
-    "-s", "EXIT_RUNTIME",
-    "-s", "ASSERTIONS",
-    "-s", "ASYNCIFY",
-    "-s", "MODULARIZE=1",
-    "-s", "EXPORT_ES6=1",
-    "-s", "SINGLE_FILE",
+    "-s",
+    "ALLOW_MEMORY_GROWTH",
+    "-s",
+    "EXIT_RUNTIME",
+    "-s",
+    "ASSERTIONS",
```

Comment:
For things like this where the two-per-line was very deliberate, will we just have to fall back on a `# wake-format off`?

Reply:
Yeah, I don't know how the formatter would ever know that this set of string parameters are special but not some other set of string parameters

Reply:
I've written/am updating an in-Wake arg parser; it's non-trivial to figure out when you're in control of the entire parser itself.  There's absolutely no way we can figure it out for arbitrary programs which may not even look like programs at the point of list definition.